Composer/packages/client/src/pages/language-generation/index.tsx

```diff
@@ -35,47 +34,30 @@ interface LGPageProps extends RouteComponentProps<{}> {
 const LGPage: React.FC<LGPageProps> = props => {
   const { state } = useContext(StoreContext);
   const { lgFiles, dialogs } = state;
+
   const path = props.location?.pathname ?? '';
   const { fileId = 'common' } = props;
   const edit = /edit(\/)*$/.test(path);
```

Comment:
a confusing name and a confusing regex here (not introduced by this PR but still a problem)
1. variable `edit` is a boolean, needs a more descriptive name
2. this regex is strange, input `edit/` or  `edit////` or `xxxxedit/` will return true, is that expected?

Reply:
@zhixzhan any idea on this? if this is not intented, can we fix in this PR

Reply:
`/\/edit(\/)?$/` is more precisely, here we expect match `/edit` `/edit/`

Reply:
Let's fix this in this PR? @liweitian  @zhixzhan